parser/proto.y

```diff
@@ -16,6 +16,7 @@ import (
 %union{
 	file         *ast.FileNode
 	syn          *ast.SyntaxNode
+	ed           *ast.EditionNode
```

Comment:
Should we update https://protobuf.com/docs/language-spec#syntax-declaration too or do you want to wait until it is officially supported here first?

Reply:
I don't want to update the language guide until editions are really usable. Right now, the only way to try them out is with `protoc --experimental_editions`, which seems a little early.

But "editions" definitely come with a fair bit of new material for the language spec.